src/styles/main.scss

```diff
@@ -1,3 +1,49 @@
-body {
-  margin: 0;
+@import "./variables.scss";
+@import "./reset.scss";
+
+.calendar {
+  display: flex;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  gap: $cellGap;
+  width: $cellSize * 7 + ($cellGap - 1) + $externalPadding * 2;
+  margin: $cellSize auto 0;
```

Comment:
BEM doesn't allow us to do so. Better stylize the parent element to center the calendar.